src/dist/conf/config.yaml

```diff
@@ -23,6 +23,8 @@ omero.server:
     # OMERO_HOME/lib/scripts
```

Comment:
`worker_pool_size` needs to be put in the example configuration. In `master` it has the form:

```
# Size of the worker pool (defaults to CPUs x 2)
#worker_pool_size: 8
```